src/managing-workflow/platform-monitoring.md

```diff
@@ -52,18 +52,30 @@ Grafana will preload several dashboards to help operators get started with monit
 These dashboards are meant as starting points and don't include every item that might be desirable to monitor in a
 production installation.
 
-Deis Workflow monitoring does not currently write data to the host filesystem or to long-term storage. If the Grafana
-instance fails, modified dashboards are lost. Until there is a solution to persist this, export dashboards and store
-them separately in version control.
+Deis Workflow monitoring by default does not write data to the host filesystem or to long-term storage. If the Grafana instance fails, modified dashboards are lost. 
+
+### On Cluster Persistence
+
+If you wish to have persistence for Grafana you can set `enabled` to `true` in the `values.yaml` file before running `helm install`.
+
+* ` grafana:`
```

Comment:
The formatting here is odd. It's a bullet point list with embedded code. Can you change it to a code snippet, like so?

```
grafana:
  # Configure the following ONLY if you want persistence for on-cluster grafana
  # GCP PDs and EBS volumes are supported only
  persistence:
    enabled: true
    size: 5Gi
```